### PR TITLE
Create robotics topic

### PR DESCRIPTION
**Pull request by andrewda**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

There are currently thousands of robotics-related repositories on GitHub, and it would be nice to provide this hub for those projects with more information for those interested in the field.

### CODE REVIEW

File: topics/robotics/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+aliases: robot
```
Comment (kenyonj) [inline topics/robotics/index.md]: It looks like the `robot` topic is used quite a bit for repositories that don't necessarily fall into this `robotics` realm. What are your thoughts on removing this alias?

File: topics/robotics/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+display_name: Robotics
+short_description: Robotics deals with the control, sensory feedback, and information
+  processing of robots.
```
Comment (kenyonj) [inline topics/robotics/index.md]:
```suggestion
```

File: topics/robotics/index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+aliases: robot
+display_name: Robotics
+short_description: Robotics deals with the control, sensory feedback, and information
```
Comment (kenyonj) [inline topics/robotics/index.md]: I think we should keep this as a one-liner
```suggestion
short_description: Robotics deals with the control, sensory feedback, and information processing of robots.
```

File: topics/robotics/index.md (hunk 1 above)
Comment (andrewda) [reply topics/robotics/index.md]: Good idea, removed!

**Pull request merged**